src/components/AddressSearch.js

```diff
@@ -105,8 +105,9 @@ const AddressSearch = (props) => {
         const {
             street_number: streetNumber,
             route: streetName,
-            locality: city,
-            sublocality: cityFallback, // Some locations only return sublocality instead of locality
+            locality: locality,
+            sublocality: sublocality,
+            postal_town: postalTown,
```

Comment:
There may be some lint errors to fix:

<img width="742" alt="image" src="https://user-images.githubusercontent.com/87341702/228204492-7531b344-7863-41f0-8d75-5aee100851be.png">

Reply:
Sorry, they should be sorted now!